tools/gen_internal_conf/src/main.rs

```diff
@@ -0,0 +1,355 @@
+extern crate clap;
+extern crate env_logger;
+extern crate log;
+extern crate regex;
+extern crate serde;
+extern crate serde_derive;
+extern crate serde_xml_rs;
+
+use clap::{App, Arg};
+use log::debug;
+use serde_derive::{Deserialize, Serialize};
+use serde_json::json;
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
+
+// This is not really used anymore.
+// Just keep it here in case we need it in the future.
+const OCCLUM_INSTANCE_DIR: &str = ".";
+
+fn main() {
+    env_logger::init();
+
+    let matches = App::new("gen_internal_conf")
+        .version("0.2.0")
+        // Input: JSON file which users may change
+        .arg(
+            Arg::with_name("user_json")
+                .short("i")
```

Comment:
The short name and the long name does not match.

Reply:
short name removed.